.github/workflows/uploadArchives.yml

```diff
@@ -0,0 +1,39 @@
+name: Upload Archives
+
+on:
+ push:
+   tags: '[0-9]+.[0-9]+.[0-9]+.*'
+
+jobs:
+ build:
+   env: 
+     ANDROID_API_LEVEL: 23
+     ANDROID_EMULATOR_LEVEL: 23
+     ANDROID_BUILD_TOOLS_VERSION: 28.0.3
+     ANDROID_ABI: armeabi-v7a
```

Comment:
not sure if this is important, but why we didn't include the arm64 support here?

Reply:
@konraddysput I removed the `ANDROID_ABI` environment variable, it's not needed here anyway.